developers.md

```diff
@@ -1,28 +1,82 @@
 # rules_pkg - Information for Developers
 
-NOTE: This is a work in progress. It is only an outline at this point.
+NOTE: This is perpetually a work in progress. We will revise it as we need.
 
-TBD: Introduction
+## PR guidelines
 
-## General
+Please discuss proposed major features and structural changes before sending a
+PR. The best way to do that is to
```

Comment:
"The best way to do that is to" perhaps should be:

"The best way to do that is to **do one or more of the following**"

Having multiple discussion forums seems a little redundant to me.

Reply:
Good change.